pages/test_analytics/test_suites.md

```diff
@@ -74,13 +74,9 @@ Once your test suite is set up, you'll have many types of information automatica
 
 For individual tests, views include trend information on reliability, test execution count, failed test execution count, and test execution duration at p50 and p95, along with detailed information about span duration and total duration of that test execution over time.
```

Comment:
I feel like this also needs to be updated, we no longer show "failed test execution count" and I don't believe we have "detailed information about span duration" on the test page?

Reply:
I agree. There is info on failures, you can toggle the tab between flakies and fails, but not a count. No span data on this page.